common/dynamicconfig/constants.go

```diff
@@ -967,6 +967,16 @@ Default is 4.`,
 after all namespace resources (i.e. workflow executions) are deleted.
 Default is 0, means, namespace will be deleted immediately.`,
 	)
+	FrontendHealthCheckFailureRate = NewGlobalFloatSetting(
+		"frontend.healthCheckFailureRat",
+		0.8,
+		`FrontendServiceHealthCheckFailureRate is the percentile of failed hosts on frontend health check`,
```

Comment:
percentile of failed hosts that do what? explain the semantics clearly

also I don't think it's a percentile, it's a percentage

Reply:
updated